src/index.ts

```diff
@@ -87,6 +87,7 @@ export { default as List } from "./components/List/List";
 export type { DescriptionProps, ListTypes } from "./components/List/List";
 export { default as Logo } from "./components/Logo/Logo";
 export type { LogoNames, LogoSizes } from "./components/Logo/Logo";
+export { default as Menu } from "./components/Menu/Menu";
```

Comment:
I _think_ you may need to export the types such as  `ListItemsData` , `ActionItem`, `GroupItem`, and `DividerItem`. When using it in a consuming app, how a dev structures their objects might be an issue if the types are not available.